### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by JayPHP**
Please don't merge - This branch is for adding tests to get some decent coverage. 

Commits will be squashed prior to merging.

### CODE REVIEW

File: tests/src/Chat/BuiltIn/UptimeTest.php [hunk 1]
```diff
@@ -52,10 +52,10 @@ public function testUptimeCommand()
             ->method('postMessage')
             ->with(
                 $this->identicalTo($this->command),
-                $this->matchingRegularExpression(self::VALID_UPTIME_EXP)
+                $this->matchesRegularExpression(self::VALID_UPTIME_EXP)
             );
 
-        yield $this->builtIn->handleCommand($this->command);
+        \amp\wait($this->builtIn->handleCommand($this->command));
```
Comment (kelunik) [inline tests/src/Chat/BuiltIn/UptimeTest.php]: `\amp\wait` → `\Amp\wait`

File: tests/src/Chat/BuiltIn/AbstractBuiltInTest.php [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?php declare(strict_types = 1);
+
+namespace Room11\Jeeves\Tests\Chat\builtIn;
```
Comment (ekinhbayar) [inline tests/src/Chat/BuiltIn/AbstractBuiltInTest.php]: `builtIn` -> `BuiltIn`

File: tests/src/Chat/BuiltIn/UptimeTest.php [hunk 3]
```diff
@@ -0,0 +1,81 @@
+<?php declare(strict_types = 1);
+
+namespace Room11\Jeeves\Tests\Chat\BuiltIn;
+
+use Amp\Success;
+use AsyncInterop\Loop;
```
Comment (ekinhbayar) [inline tests/src/Chat/BuiltIn/UptimeTest.php]: Unutilized

File: composer.json [hunk 4]
```diff
@@ -42,10 +42,10 @@
         "room11/dom-utils": "~1.0",
         "room11/google-searcher": "~1.0",
         "sabre/uri": "^1.0",
-        "sebastian/version": "^1.0",
-        "symfony/yaml": "^3.0"
+        "sebastian/version": "^2.0",
+        "symfony/yaml": "^3.0",
+        "phpunit/phpunit": "^6.0"
```
Comment (krakjoe) [inline composer.json]: phpunit is typically require-dev

**Comment by JayPHP** [comment]
Accidental merge, valid anyway.

**Pull request merged**